# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../pkgconfig/pkg-config-0.23-sysroot.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

https://bugs.freedesktop.org/attachment.cgi?id=21210

--- a/pkg.c	2008-01-16 13:59:49.000000000 -0800
+++ b/pkg.c	2008-12-16 10:23:15.000000000 -0800
@@ -472,9 +472,8 @@
   while (tmp != NULL)
     {
       char *tmpstr = (char*) tmp->data;
-      if (pcsysrootdir != NULL)
-	{
-	  if (tmpstr[0] == '-' &&
+      if (pcsysrootdir != NULL &&
+	      tmpstr[0] == '-' &&
 	      (tmpstr[1] == 'I' ||
 	       tmpstr[1] == 'L'))
 	    {
@@ -483,7 +482,6 @@
 	      g_string_append (str, pcsysrootdir);
 	      g_string_append (str, tmpstr+2);
 	    }
-	}
       else 
 	{
 	  g_string_append (str, tmpstr);
